Exonoesis/Helping-Hand
# Updates

## What is the purpose of these changes?
Bevy had an update and our other third-party libraries also updated in response. Keeping up to date is essential.

## What has changed?
Bevy, ECS LdTK, and Kira Audio have all been updated to the latest version and the code has been refactored as part of the migration.

## How can I verify this works?
1. Run Unit Tests -> See that they show no errors.
2. Launch the program -> Quick visual inspection will show that the Main Menu is without errors
3. Click the Play Button (and Quit if you wish) -> See that the game loads without error
4. Use "WASD" keys to move the character -> See that the game behaves as it did before the updates with no errors
